scipy/spatial/transform/_rotation.pyx

```diff
@@ -15,12 +15,20 @@ np.import_array()
 
 # utilities for empty array initialization
 cdef inline double[:] _empty1(int n) noexcept:
+    if n == 0:
+        return array(shape=(1,), itemsize=sizeof(double), format=b"d")[:0]
     return array(shape=(n,), itemsize=sizeof(double), format=b"d")
 cdef inline double[:, :] _empty2(int n1, int n2) noexcept :
+    if n1 == 0:
+        return array( shape=(1, n2), itemsize=sizeof(double), format=b"d" )[:0]
```

Comment:
Odd space.

Reply:
Due to the empty-placeholder, this allowed to include 0-dimensional arrays with minimal change

Reply:
I meant why put additional spaces near the arguments in `array`? Just a minor formatting thing.

Reply:
Extra spaces before `shape=(1, n2)` and after `format=b"d"`.